backend/src/cron/tasks.ts

```diff
@@ -109,12 +110,17 @@ export const transactionCountHistoryCheck: RegularCheckFn = {
   },
 };
 
+export const tokensSupplyCheck: RegularCheckFn = {
+  description: "circulatingSupplyCheck",
+  fn: publishOnChange("tokensSupply", queryTokensSupply, () => {
+    const endOfDayTimestamp = endOfDay(new Date()).valueOf();
+    return endOfDayTimestamp - Date.now();
```

Comment:
@frol I guess "end of the day" is not the right heuristics for the update interval.
Currently database lags for 2 days so we don't stumble upon showing incomplete data for say today, but could we in the future?

Reply:
Can we just update it hourly? Circulating supply is getting computed for the day before when it is a few minutes in after midnight UTC.I believe it is totally fine if we lag by 1 hour.

Reply:
I guess we can